starter-code/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { NotFoundComponent } from './not-found/not-found.component';
+import { MyHomeComponentComponent } from './my-home-component/my-home-component.component';
+import { MyMovieComponentComponent } from './my-movie-component/my-movie-component.component';
+
+
+
+const routes: Routes = [
+  {path: 'home', component: MyHomeComponentComponent },
```

Comment:
This my only suggested improvement
```js   
{ path: "", component: MyHomeComponentComponent }, 
```
Thanks to this, your home will be displayed at first glance :)